templates/songs.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang=en>
+<head>
+<meta http-equiv=content-type content="text/html; charset=utf-8" />
+<meta name=viewport content="width=device-width, initial-scale=1">
+<link rel=icon type=image/x-icon href=/static/hacky-day.png />
+<link rel=stylesheet type=text/css href=/static/index.css />
+
+<title>hack·star - Song Management</title>
+<script>
+function deleteSong(songId) {
+  if (confirm('Are you sure you want to delete this song? This action cannot be undone.')) {
+    fetch('/songs/' + songId + '/delete', {
+      method: 'DELETE'
+    }).then(response => {
+      if (response.ok) {
+        location.reload();
+      } else {
+        alert('Error deleting song');
+      }
+    }).catch(error => {
+      alert('Error deleting song: ' + error);
+    });
+  }
+}
+</script>
+</head>
+<body>
+
+	<header class=top>
+		<a href=/><img src=/static/hacky-day.png alt='hacky·day' /></a>
+    Song Management
+	</header>
+
+	<main>
+    <div id="songs-container">
+      <h2>Songs ({{ songs|length }})</h2>
+      
+      {% if songs %}
+      <div id="songs-content">
+        {% for song in songs %}
+        <div class="song-item">
+          <div class="song-metadata">
+            {% if song.cover %}
+            <div class="song-cover">
+              <img src="{{ song.cover }}" alt="Cover" />
+            </div>
+            {% endif %}
+            <div class="song-info">
```

Comment:
When hitting delete, instead of reloading the page, can we instead add an ID to the song DOM element like this:
```suggestion
            <div class="song-info" id="song-{{ song.hex_id }}">
```
And then use `document.getElementById` in JS to get and then remove the element without reloading the page?